Typos

OlegAlexander opened the pull request:
## Summary

@BillWagner, thank you very much for approving my PRs so far! 

In addition to the minor typos on this page, I also wanted to update the `toPigLatin` function to use the F# 6 indexing syntax without the dot. (`expr[idx]` instead of `expr.[idx]`) However, the code snippet is [a link that I don't have access to](https://github.com/dotnet/docs/blob/main/docs/fsharp/get-started/~/samples/snippets/fsharp/getting-started/to-pig-latin.fsx). 

Please consider updating the `toPigLatin` example to the new `expr[idx]` syntax on your end. If you decide to do so, please keep in mind that the example appears 4 times on this page:

1. Under _Write your first script_, the example uses the old `expr.[idx]` syntax.
2. The screenshot of sending the function to the FSI process.
3. Under _Explaining the code_, point 2 `if..then..else` expression.
4. Under _Turn the console app into a Pig Latin generator_, in `module PigLatin` the example uses the NEW `expr[idx]` syntax! 

Finally, please let me know if you _do not_ want me to send you PRs related to the indexing syntax in the future. Thanks!

OlegAlexander committed:
Typos

IEvangelist commented:
Hi @OlegAlexander - thank you for this PR, it is much appreciated. As for the file you're looking for, it is here:

https://github.com/dotnet/docs/blob/main/samples/snippets/fsharp/getting-started/to-pig-latin.fsx

You could update it as part of this PR, or create a new PR. Just let me know.

OlegAlexander committed:
Using F# 6 indexing syntax

dnfadmin commented:
[![CLA assistant check](https://cla.dotnetfoundation.org/pull/badge/not_signed)](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=29647) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=29647) before we can accept your contribution.<br/><br/>:x: OlegAlexander [sign now](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=29647)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/docs?pullRequest=29647) it.</sub>

OlegAlexander committed:
Replace FSI screenshot using F# 6 indexing syntax

OlegAlexander commented:
Hello @IEvangelist, thanks for pointing me to the correct file. I've updated the `toPigLatin` function to use the new indexing syntax. Under _Explaining the code_, point 2, `if..then..else` expression, it looks like the link is still pointing to the same fsx file, lines 8-11, so that should be fine.

I've also updated the FSI screenshot showing the new indexing syntax. (Not sure if it needs to be resized--my screenshot is somewhat bigger than the old one but otherwise very similar.) 

Please let me know if everything is in order. Thank you!
